jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/ProjectLabelEnricher.java

```diff
@@ -160,8 +161,13 @@ public void visit(ObjectMetaBuilder element) {
         if (enableProjectLabel) {
             ret.put("project", groupArtifactVersion.getArtifactId());
         } else {
-            // default label is app
-            ret.put("app", groupArtifactVersion.getArtifactId());
+        	String customAppName = Configs.asString(getConfig(Config.CUSTOM_APP_NAME));
+        	if (customAppName != null && !customAppName.isEmpty()) {
+        		ret.put("app", customAppName);
+        	} else {
+                // default label is app
+                ret.put("app", groupArtifactVersion.getArtifactId());
+        	}
```

Comment:
I think this can be safely replaced by
```suggestion
/* ... */
/* L58 */ CUSTOM_APP_NAME("customAppName", null);
/* ... */
                ret.put("app", getConfig(Config.CUSTOM_APP_NAME, groupArtifactVersion.getArtifactId());
```